cfme/common/vm.py

```diff
@@ -717,6 +725,69 @@ def set_retirement_date(self, when, warn=None):
             sel.click(form_buttons.save)
 
 
+    def equal_drift_results(self, row_text, section, *indexes):
+        """ Compares drift analysis results of a row specified by it's title text
+
+        Args:
+            row_text: Title text of the row to compare
+            section: Accordion section where the change happened; this section must be activated
+            indexes: Indexes of results to compare starting with 0 for first row (latest result).
+                     Compares all available drifts, if left empty (default).
+
+        Note:
+            There have to be at least 2 drift results available for this to work.
+
+        Returns:
+            ``True`` if equal, ``False`` otherwise.
+        """
+        # mark by indexes or mark all
+        self.load_details(refresh=True)
+        sel.click(InfoBlock("Properties", "Drift History"))
+        if indexes:
+            drift_table.select_rows_by_indexes(*indexes)
+        else:
+            # We can't compare more than 10 drift results at once
+            # so when selecting all, we have to limit it to the latest 10
+            if len(list(drift_table.rows())) > 10:
+                drift_table.select_rows_by_indexes(*range(0, min(10, len)))
+            else:
+                drift_table.select_all()
+        tb.select("Select up to 10 timestamps for Drift Analysis")
+
+        # Make sure the section we need is active/open
+        sec_loc_map = {
+            'Properties': 'Properties',
+            'Security': 'Security',
+            'Configuration': 'Configuration',
+            'My Company Tags': 'Categories'}
+        sec_loc_template = "//div[@id='all_sections_treebox']//li[contains(@id, 'group_{}')]" \
+                           "//span[contains(@class, 'dynatree-checkbox')]"
+        sec_checkbox_loc = "//div[@id='all_sections_treebox']//li[contains(@id, 'group_{}')]" \
+            "//span[contains(@class, 'dynatree-checkbox')]".format(sec_loc_map[section])
+        sec_apply_btn = "//div[@id='accordion']/a[contains(normalize-space(text()), 'Apply')]"
+
+        # Deselect other sections
+        for other_section in sec_loc_map.keys():
+            other_section_loc = sec_loc_template.format(sec_loc_map[other_section])
+            other_section_classes = sel.get_attribute(other_section_loc + '/..', "class")
+            if other_section != section and 'dynatree-partsel' in other_section_classes:
+                # Element needs to be checked out if it has no dynatree-selected
+                if 'dynatree-selected' not in other_section_classes:
+                    sel.click(other_section_loc)
+                sel.click(other_section_loc)
+
+        # Activate the required section
+        sel.click(sec_checkbox_loc)
+        sel.click(sec_apply_btn)
+
+        if not tb.is_active("All attributes"):
+            tb.select("All attributes")
+        d_grid = DriftGrid()
+        if any(d_grid.cell_indicates_change(row_text, i) for i in range(0, len(indexes))):
```

Comment:
`indexes` is checked earlier in the function, and I don't see it set afterwards - this will raise an exception if indexes is None.